### ISSUE TITLE
[question] How to make more mobile friendly

### ISSUE THREAD

**Issue by InAnimaTe** [sethmacleod/aerial#6]
First off, totally digging this theme. Probably one of the best personal splash pages I've ever seen. I do have one question on modifying this to look a bit better for certain situations.

How can I tune this to display nicely on mobile and in thin browser sizing modes. The screenshot below is how it looks when I half-size my browser window:
![screenshot from 2017-05-19 17-40-08](https://cloud.githubusercontent.com/assets/1192726/26267878/5b81584a-3cba-11e7-92b2-dbb1288cb15f.png)

Instead, can I have something where maybe only portion of the image is shown but takes up the full background? Or something where a different image is used at a certain resolution etc..

tl;dr I don't want people to see the black background unless the image isn't actually available. 

Thanks!

**Comment by sethmacleod** [comment]
Hi @InAnimaTe 

Sorry for taking so long to respond. I'm glad you enjoy the theme by [HTML5 UP](https://html5up.net/). When they created the theme, they designed it for an image width of 1500px. Their background image is 1500x500. What are the dimensions of your background image?

You can check out a demo on their site [here](https://html5up.net/aerial) for what it should look like at the different sizes.
